# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/cinelerra-gg/hotfix-libzmpeg3.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- cinelerra-5.1/libzmpeg3/Makefile.vanilla	2026-01-31 11:02:25.788801533 +0000
+++ cinelerra-5.1/libzmpeg3/Makefile	2026-01-31 11:03:15.632805149 +0000
@@ -79,8 +79,8 @@
 UTILS += $(OBJDIR)/zmpeg3cc2txt
 
 LIBS = -lm -lpthread
-ifeq ($(WANT_CIN_3RDPARTY),no)
-LIBS += -la52
+ifneq ($(WANT_CIN_3RDPARTY),no)
+LIBS += -la52 `pkg-config --libs liba52`
 endif
 ifeq ($(shell uname -o), Android)
 LIBS += ../libbthread-master/jpt.a
